examples/cypress/BUILD.bazel

```diff
@@ -0,0 +1,34 @@
+load("@cypress//:index.bzl", "cypress_web_test")
+load("@npm//@bazel/typescript:index.bzl", "ts_library")
+
+ts_library(
+    name = "plugins_file",
+    testonly = True,
+    srcs = ["plugin.ts"],
+    tsconfig = ":tsconfig.json",
+    deps = [
+        "@npm//@types/node",
+        "@npm//express",
+    ],
+)
+
+ts_library(
```

Comment:
any particular reason you chose ts_library rather than ts_project? I expect most users will prefer the latter since it's compatible with their existing code

Reply:
The reason I opted for ts_library was to test that I consumed the ts_library providers correctly. Maybe I should use both a ts_library and ts_project? What do you think?

Reply:
we shouldn't have a difference between providers, such that you can tell whether ts_project or ts_library was used to compile your dependency. Could you file an issue with the details about what's different? I hope you don't still need the "typescript" provider